smarts/core/trap_manager.py

```diff
@@ -240,7 +240,11 @@ def traps(self):
     @staticmethod
     def _hijack_vehicle(sim, vehicle_id, agent_id, mission):
         agent_interface = sim.agent_manager.agent_interface_for_agent_id(agent_id)
-        planner = MissionPlanner(sim.scenario.waypoints, sim.scenario.road_network)
+        planner = MissionPlanner(
```

Comment:
This is fine for now, I will give myself a ticket to clean up the `TrapManager` logic.